src/chart/draw/consolidateData/transformData/makeNest.js

```diff
@@ -0,0 +1,174 @@
+import { nest, scale, extent, range, ascending, quantile, sum, merge, descending } from 'd3';
```

Comment:
Assuming this just a copy/paste and not a significant refactor?

Reply:
Yeah, I'm afraid to touch it.